src/content/docs/iast/install.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Install IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Installation
+metaDescription: Learn how to install New Relic IAST.
+freshnessValidatedDate: never
+---
+
+import iastSetup from 'images/iast_screenshot-full_install.webp'
+
+import iastInstall from 'images/iast_screenshot-crop_install.webp'
+
+<Callout variant="important">
+    Run IAST with non-production deployments only.
+
+    Why? IAST tests by invoking HTTP requests with an exploit payload. You must use IAST in non-production environments with only simulated data to avoid both data corruption and introducing exploits into your live code. In addition, when you run IAST on new code in pre-production, you catch potential vulnerabilities before they go live.
+
+</Callout>
+
+To install New Relic IAST:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Set up IAST with applications**.
+
+    <img
+        title="Set up IAST with applications"
+        alt="Set up IAST with applications"
+        src={iastSetup}
+    />
+
+2. Confirm IAST is not running in a production environment and click **Continue**.
+
+3. Select the language of your application and complete the steps.
+
+    <img
+        title="Install New Relic IAST"
+        alt="Install New Relic IAST"
+        src={iastInstall}
+    />
+
+4. Make sure your `newrelic.yml` config file is updated as follows:
+
+    ```yml
+    host: staging-collector.newrelic.com
+    security:
+        enabled: true
+    agent:
+        enabled: true
+    validator_service_url: wss://csec-staging.nr-data.net
+    ```
+
+5. Once you've completed all the steps, restart your application and generate traffic against the application's APIs.
+
+6. Click **See your data** to see an overview of your tested application.
+
+
+For more detailed instructions, check out these examples demo below.
+
+See this example demo about how to set up IAST with Java (approx. 2:48 minutes):
```

Comment:
Is it possible to put the examples in a collapser? Right now they're a little awkward on the page and I think they could be nested into a collapser to make it more visually appealing.